lib/vu_state.go

```diff
@@ -34,9 +34,8 @@ type State struct {
 	Options        Options
 	BuiltinMetrics *metrics.BuiltinMetrics
 
-	// Logger. Avoid using the global logger.
-	// TODO: change to logrus.FieldLogger when there is time to fix all the tests
-	Logger *logrus.Logger
+	// Logger instance for every VU, avoid using the global logger.
```

Comment:
```suggestion
	// Logger instance for every VU.
```
nit: Is it relevant for this part of the code? This is the unique logger that can be invoked, and doing a Print will trigger the linter.